test/sys/test_resource.rs

```diff
@@ -10,6 +10,7 @@ use nix::sys::resource::{getrusage, UsageWho};
 /// to put the new soft limit in effect, and then getrlimit() once more to ensure the limits have
 /// been updated.
 #[test]
+#[cfg_attr(target_os = "cygwin", ignore)]
```

Comment:
Why is this test ignored? looks like the symbol it is testing is available on Cygwin

Reply:
`getrlimit` always returns const value for `RLIMIT_NOFILE`.

https://github.com/cygwin/cygwin/blob/8e50c7af7c49819245739d6f626f6fecc394ef7f/winsup/cygwin/resource.cc#L278